lms/static/js/demographics_collection/DemographicsCollectionBanner.jsx

```diff
@@ -1,54 +1,96 @@
 /* global gettext */
 import React from 'react';
+import Cookies from 'js-cookie';
 import { DemographicsCollectionModal } from './DemographicsCollectionModal';
+import AxiosJwtTokenService from '../jwt_auth/AxiosJwtTokenService';
 
 export class DemographicsCollectionBanner extends React.Component {
 
   constructor(props) {
     super(props);
     this.state = {
       modalOpen: false,
+      hideBanner: false
+    }
+
+    // Get JWT token service to ensure the JWT token refreshes if needed
+    const accessToken = this.props.jwtAuthToken;
+    const refreshUrl = `${this.props.lmsRootUrl}/login_refresh`;
+    this.jwtTokenService = new AxiosJwtTokenService(
+      accessToken,
+      refreshUrl,
+    );
+
+    this.dismissBanner = this.dismissBanner.bind(this);
+  }
+
+  async dismissBanner(e) {
+    // The dismiss button is nested inside an <a>, so we need to call stopPropogation here to
+    // keep the Modal from also opening when the Dismiss button is clicked
+    e.stopPropagation();
+
+    const requestOptions = {
+      method: 'PATCH',
+      credentials: 'include',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRFTOKEN': Cookies.get('csrftoken'),
+      },
+      body: JSON.stringify({
+        show_call_to_action: false,
+      })
+    };
+
+    let response = {};
+    await this.jwtTokenService.getJwtToken();
```

Comment:
I meant to put a TODO here to remove this line before the PR was opened, but I missed it. I don't think I need this here so I can remove before merging (pending approval)